cmd/kpod/images.go

```diff
@@ -55,155 +84,245 @@ func imagesCmd(c *cli.Context) error {
 	if err := validateFlags(c, imagesFlags); err != nil {
 		return err
 	}
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "Could not get config")
-	}
-	store, err := getStore(config)
+
+	runtime, err := getRuntime(c)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "Could not get runtime")
 	}
+	defer runtime.Shutdown(false)
 
-	quiet := false
-	if c.IsSet("quiet") {
-		quiet = c.Bool("quiet")
-	}
-	noheading := false
-	if c.IsSet("noheading") {
-		noheading = c.Bool("noheading")
-	}
-	truncate := true
-	if c.IsSet("no-trunc") {
-		truncate = !c.Bool("no-trunc")
-	}
-	digests := false
-	if c.IsSet("digests") {
-		digests = c.Bool("digests")
-	}
-	outputFormat := genImagesFormat(quiet, truncate, digests)
+	format := genImagesFormat(c.Bool("quiet"), c.Bool("noheading"), c.Bool("digests"))
```

Comment:
Possible nit.  Maybe make this an else for the 'if' at line 95?

Reply:
fixed.